aws-kendra-index/src/main/java/software/amazon/kendra/index/DeleteHandler.java

```diff
@@ -31,7 +35,7 @@ public class DeleteHandler extends BaseHandlerStd {
             // Set the delay to one minute so the stabilization code only calls
             // DescribeIndex every minute - delete can take a few minutes
             // so there's no need to check the index has been deleted more than once a minute.
-            .delay(Duration.ofMinutes(1))
+            .delay(Duration.ofMinutes(5))
```

Comment:
Can you update the comments?

Reply:
I updated this to two minutes and updated the comments.